ceph/ceph.py

```diff
@@ -1097,6 +1097,58 @@ def get_nodes_in_location(self, location: str) -> list:
         """Return the list of nodes found in the location."""
         return [node for node in self.node_list if node.vm_node.location == location]
 
+    def get_node_ip_by_hostname(self, hostname):
+        """
+
+        Args:
+            hostname:
+
+        Returns:
+
+        """
+        cmd = "hostname -I | awk '{print $1}'"
+        node_list = [node for node in self.get_nodes() if node.hostname == hostname]
+        node = node_list[0] if len(node_list) > 0 else None
+        return node.exec_command(cmd=cmd)[0].replace("\n", "")
```

Comment:
Does it makes sense to add ip address to node dict similar to hostname rather than fetching it each time by ssh?

Reply:
Yes true it makes sense, I will keep this point and raise another small PR to update this, after testing out the changes properly. @VasishtaShastry would that work?

Reply:
Yes

Reply:
@manasagowri we aready have ip address method in node object we can use the same,
refer: https://github.com/red-hat-storage/cephci/blob/43ed6426d7741862409814cf81d2812fe2eeb79d/ceph/ceph_admin/dashboard.py#L165